java/list-kml-contents/README.md

```diff
@@ -14,10 +14,10 @@ The contents of the KML file are shown in a tree. Select a node to zoom to that
 
 ## How it works
 
-1. The KML file is added to the scene as a layer.
-2. Root nodes from `KmlDataset.RootNodes` is recursively explored to create a drill down menu.
+1. Add the KML file to the scene as a layer.
+2. Explore the root nodes of the `KmlDataset` recursively explored to create a view model.
```

Comment:
something funny with the english here, I'm not sure how best to change it - not sure my suggestion makes sense? 
```suggestion
2. Explore the root nodes of the `KmlDataset` which are recursively explored to create a view model.
```

Reply:
yes, I thought it could have been a typo and what was meant was "recursively exploded".